data/autoyast_sle12/create_hdd/create_hdd_ha_sles.xml.ep

```diff
@@ -305,25 +305,66 @@
         <products config:type="list">
             <product><%= uc $get_var->('SLE_PRODUCT') %></product>
         </products>
-        <!-- Workaround for bsc#1202234: [addon]-release packages are missing after autoyast installation. -->
         <packages config:type="list">
-        % foreach (values %$addons) {
+            <!-- Workaround for bsc#1202234: [addon]-release packages are missing after autoyast installation. -->
+            % foreach (values %$addons) {
             <package><%= lc($_->{name}) %>-release</package>
-        % }
+            % }
+            <!-- end of workaround -->
+            <package>sles-release</package>
```

Comment:
I cannot tell for the commit or any message in the PR if this is some intermediate test (that can be signal in many ways, as we trigger review even if it is WIP).

Anyway this patterns and services are very specific, we don't need to include them in sp4, we could could just create another profile, if they would be a few, perhaps with some `if` would do, but doesn't look like it worth it to have all in the same file (but perhaps this is just a test as I mentioned above).